### PR TITLE
Replace console.log with LogOutputChannel log

### PR DESCRIPTION
**Pull request by JasonYeMSFT**

### CODE REVIEW

File: src/chat/copilotInteractions.ts [hunk 1]
```diff
@@ -40,7 +41,7 @@ function debugCopilotInteraction(responseStream: vscode.ChatResponseStream, msg:
     if (showDebugCopilotInteractionAsProgress) {
         responseStream.markdown(new vscode.MarkdownString(`\n\n${new Date().toISOString()} >> \`${msg.replace(/\n/g, "").trim()}\`\n\n`));
     }
-    console.log(`${new Date().toISOString()} >> \`${msg.replace(/\n/g, "").trim()}\``);
+    ext.outputChannel.info(`${new Date().toISOString()} >> \`${msg.replace(/\n/g, "").trim()}\``);
```
Comment (bwateratmsft) [inline src/chat/copilotInteractions.ts:44]: Date is automatically prepended to log output channel so we can remove it from here. Also, I'd suggest raising this to verbose but I'm open to debate on that.

File: src/chat/extensions/AzureExtension.ts [hunk 2]
```diff
@@ -56,7 +57,7 @@ export class AzureExtension {
             return (await this._getCommandConfigs())
                 .filter((commandConfig): commandConfig is SimpleCommandConfig => commandConfig.type === "simple");
         } catch (error) {
-            console.log(`Error getting wizard commands from ${this.extensionDisplayName} extension: ${JSON.stringify(error)}`);
+            ext.outputChannel.info(`Error getting wizard commands from ${this.extensionDisplayName} extension: ${JSON.stringify(error)}`);
```
Comment (bwateratmsft) [inline src/chat/extensions/AzureExtension.ts:60]:
```suggestion
            ext.outputChannel.info(`Error getting simple commands from ${this.extensionDisplayName} extension: ${JSON.stringify(error)}`);
```

File: src/chat/extensions/AzureExtension.ts [hunk 3]
```diff
@@ -66,7 +67,7 @@ export class AzureExtension {
             return (await this._getCommandConfigs())
                 .filter((commandConfig): commandConfig is SkillCommandConfig => commandConfig.type === "skill");
         } catch (error) {
-            console.log(`Error getting wizard commands from ${this.extensionDisplayName} extension: ${JSON.stringify(error)}`);
+            ext.outputChannel.info(`Error getting wizard commands from ${this.extensionDisplayName} extension: ${JSON.stringify(error)}`);
```
Comment (bwateratmsft) [inline src/chat/extensions/AzureExtension.ts:70]:
```suggestion
            ext.outputChannel.info(`Error getting skill commands from ${this.extensionDisplayName} extension: ${JSON.stringify(error)}`);
```

File: src/chat/extensions/AzureExtension.ts [hunk 4]
```diff
@@ -144,7 +145,7 @@ export class AzureExtension {
             return this._cachedCommandConfigs;
         } catch (error) {
             this._cachedCommandConfigs = [];
-            console.log(`Error getting wizard commands from ${this.extensionDisplayName} extension: ${JSON.stringify(error)}`);
+            ext.outputChannel.info(`Error getting wizard commands from ${this.extensionDisplayName} extension: ${JSON.stringify(error)}`);
```
Comment (bwateratmsft) [inline src/chat/extensions/AzureExtension.ts:148]:
```suggestion
            ext.outputChannel.info(`Error getting commands from ${this.extensionDisplayName} extension: ${JSON.stringify(error)}`);
```

File: src/chat/extensions/AzureExtension.ts [hunk 5]
```diff
@@ -126,7 +127,7 @@ export class AzureExtension {
             return this._cachedAgentBenchmarkConfigs;
         } catch (error) {
             this._cachedAgentBenchmarkConfigs = [];
-            console.log(`Error getting wizard commands from ${this.extensionDisplayName} extension: ${JSON.stringify(error)}`);
+            ext.outputChannel.info(`Error getting wizard commands from ${this.extensionDisplayName} extension: ${JSON.stringify(error)}`);
```
Comment (bwateratmsft) [inline src/chat/extensions/AzureExtension.ts:130]:
```suggestion
            ext.outputChannel.info(`Error getting benchmarks from ${this.extensionDisplayName} extension: ${JSON.stringify(error)}`);
```

File: src/chat/intentDetection.ts [hunk 6]
```diff
@@ -85,9 +86,9 @@ export async function detectIntent(targets: IntentDetectionTarget[], request: Ag
     const typeChatResult = await detectIntentTypeChat(targets, request);
 
     // Log both natural language results and typeChat results for comarison.
-    console.log("Natural Language intent detection result: ", naturalLanguageResult);
-    console.log("TypeChat intent detection result: ", typeChatResult);
-    console.log("Result match", naturalLanguageResult?.name === typeChatResult?.name);
+    ext.outputChannel.info("Natural Language intent detection result: ", naturalLanguageResult);
+    ext.outputChannel.info("TypeChat intent detection result: ", typeChatResult);
+    ext.outputChannel.info("Result match", naturalLanguageResult?.name === typeChatResult?.name);
```
Comment (bwateratmsft) [inline src/chat/intentDetection.ts:91]: If we're raising the chat results to verbose, we should also raise this to verbose.

File: src/chat/copilotInteractions.ts (hunk 1 above)
Comment (JasonYeMSFT) [reply src/chat/copilotInteractions.ts]: I removed the date from the log message and adjusted the log level to debug. In VSCode, "debug" is right above the "info" level.

File: src/chat/extensions/AzureExtension.ts (hunk 2 above)
Comment (JasonYeMSFT) [reply src/chat/extensions/AzureExtension.ts]: Fixed.

File: src/chat/extensions/AzureExtension.ts (hunk 3 above)
Comment (JasonYeMSFT) [reply src/chat/extensions/AzureExtension.ts]: Fixed.

File: src/chat/intentDetection.ts (hunk 6 above)
Comment (JasonYeMSFT) [reply src/chat/intentDetection.ts]: I raised these logging to "debug" level.

File: src/chat/extensions/AzureExtension.ts (hunk 5 above)
Comment (JasonYeMSFT) [reply src/chat/extensions/AzureExtension.ts]: Fixed.

File: src/chat/extensions/AzureExtension.ts (hunk 4 above)
Comment (JasonYeMSFT) [reply src/chat/extensions/AzureExtension.ts]: Fixed.

**Review by bwateratmsft — APPROVED** [review]

**Pull request merged**